modules/ting_search/modules/subsearch_kpi/subsearch_kpi.info

```diff
@@ -0,0 +1,7 @@
+name = Subsearch KPI - What Did You Mean?
+description = Extend Ting Search with secondary search
+package = Ding!
+core = 7.x
+version = 0.1
```

Comment:
We could add a configuration hint here:

`configure = admin/config/ting/settings`